Doxyfile

```diff
@@ -1113,7 +1229,7 @@ GENERATE_HTML          = YES
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            = site/reference
+HTML_OUTPUT            = docs/reference
```

Comment:
Pardon my partial lack of comprehension with regards to the changes you made to the build process. If I'm getting this correctly, building to `docs` will cause mkdocs to automatically include the reference in the final site folder, is that correct?

Reply:
I'm guessing this also explains the removal of the second `../`.